react-native-in-app-payments-quickstart/app/components/CardsOnFileModal.tsx

```diff
@@ -19,25 +19,24 @@ import {
   StyleSheet,
   Text,
 } from 'react-native';
-import PropTypes from 'prop-types';
 
 import CardsOnFileTitleView from './CardsOnFileTitleView';
 import CardsOnFileCardView from './CardsOnFileCardView';
 import GreenButton from './GreenButton';
 
-CardsOnFileModal.propTypes = {
-  cardsOnFile: PropTypes.array.isRequired,
-  onSelectCardOnFile: PropTypes.func.isRequired,
-  onCloseCardsOnFileScreen: PropTypes.func.isRequired,
-  onShowCustomerCardEntry: PropTypes.func.isRequired,
+interface Props {
+  onCloseCardsOnFileScreen: any,
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.